src/whylogs/util/dsketch.py

```diff
@@ -211,7 +211,7 @@ def deserialize(x: bytes):
         If x is an empty sketch, None is returned
         """
         if len(x) <= 8:
-            FrequentItemsSketch()
+            return FrequentItemsSketch()
```

Comment:
@ibackus I think this was the bug in the existing code?

Reply:
That definitely seems to be a problem.